docs/protocol/concepts/tokens.md

```diff
@@ -15,14 +19,14 @@ There are three types of assets:
 
 This matches Ethereum denomination of:
 
-`1 ETH = 10<sup>18</sup> wei`
+1 ETH = 10<sup>18</sup> wei
```

Comment:
we should use consistent style between these those formulas

<img width="317" alt="image" src="https://user-images.githubusercontent.com/42640438/220898581-da63c10b-5297-4067-8c97-25b5a4822a42.png">